### PR TITLE
#772; removes passwords from logs.

### PR DESCRIPTION
**Pull request by a-murphy**
#772 

Removes all of the logged passwords I could find and replaces the database password and installer keys with asterisks in the confirmation after they are entered.

### CODE REVIEW

File: common/scripts/boot_service.sh [hunk 1]
```diff
@@ -10,8 +10,6 @@ __validate_service_configs() {
   __process_msg "Service $SERVICE_NAME configuration"
   __process_msg "SERVICE: $SERVICE_NAME"
   __process_msg "SERVICE_IMAGE: $SERVICE_IMAGE"
-  __process_msg "ACCESS_KEY: $ACCESS_KEY"
```
Comment (ric03uec) [inline common/scripts/boot_service.sh]: lets just print the number of characters in the key here

File: common/scripts/docker/startFakeAPI.sh [hunk 2]
```diff
@@ -14,13 +14,10 @@ __validate_api_envs() {
   __process_msg "API_IMAGE: $API_IMAGE"
   __process_msg "DBNAME: $DBNAME"
   __process_msg "DBUSERNAME: $DBUSERNAME"
-  __process_msg "DBPASSWORD: $DBPASSWORD"
   __process_msg "DBHOST: $DBHOST"
   __process_msg "DBPORT: $DBPORT"
   __process_msg "DBDIALECT: $DBDIALECT"
   __process_msg "LOGS_FILE: $LOGS_FILE"
-  __process_msg "ACCESS_KEY: $ACCESS_KEY"
```
Comment (ric03uec) [inline common/scripts/docker/startFakeAPI.sh]: print key length

File: common/scripts/lib/_helpers.sh [hunk 3]
```diff
@@ -223,7 +223,6 @@ __print_runtime() {
   __process_msg "DB_IP: $DB_IP"
   __process_msg "DB_PORT: $DB_PORT"
   __process_msg "DB_USER: $DB_USER"
-  __process_msg "DB_PASSWORD: $DB_PASSWORD"
```
Comment (ric03uec) [inline common/scripts/lib/_helpers.sh]: print length of key

File: common/scripts/installWorker.sh [hunk 4]
```diff
@@ -19,8 +19,6 @@ __validate_worker_envs() {
   __process_msg "MASTER_HOST: $MASTER_HOST"
   __process_msg "MASTER_PORT: $MASTER_PORT"
   __process_msg "WORKER_JOIN_TOKEN: $WORKER_JOIN_TOKEN"
-  __process_msg "ACCESS_KEY: $ACCESS_KEY"
```
Comment (ric03uec) [inline common/scripts/installWorker.sh]: print length of key

File: common/scripts/installVault.sh [hunk 5]
```diff
@@ -22,7 +22,6 @@ __validate_vault_envs() {
   __process_msg "DBPORT: $DBPORT"
   __process_msg "DBNAME: $DBNAME"
   __process_msg "DBUSERNAME: $DBUSERNAME"
-  __process_msg "DBPASSWORD: $DBPASSWORD"
```
Comment (ric03uec) [inline common/scripts/installVault.sh]: print length of key

File: common/scripts/installState.sh [hunk 6]
```diff
@@ -19,7 +19,6 @@ __validate_state_envs() {
   __process_msg "STATE_DATA_DIR: $STATE_DATA_DIR"
   __process_msg "STATE_CONFIG_DIR: $STATE_CONFIG_DIR"
   __process_msg "STATE_HOST: $STATE_HOST"
-  __process_msg "STATE_PASS: $STATE_PASS"
```
Comment (ric03uec) [inline common/scripts/installState.sh]: print length of key

File: common/scripts/installMsg.sh [hunk 7]
```diff
@@ -19,9 +19,7 @@ __validate_msg_envs() {
   __process_msg "MSG_CONFIG_DIR: $MSG_CONFIG_DIR"
   __process_msg "MSG_HOST: $MSG_HOST"
   __process_msg "MSG_UI_USER: $MSG_UI_USER"
-  __process_msg "MSG_UI_PASS: $MSG_UI_PASS"
   __process_msg "MSG_USER: $MSG_USER"
-  __process_msg "MSG_PASS: $MSG_PASS"
```
Comment (ric03uec) [inline common/scripts/installMsg.sh]: print length of pass

**Comment by a-murphy** [comment]
Updated to print the number of characters in the place of the removed keys and passwords.

**Pull request merged**